src/BUILD

```diff
@@ -179,32 +179,52 @@ filegroup(
     visibility = ["//src/test/shell/bazel:__pkg__"],
 )
 
+cc_binary(
```

Comment:
nit: can we leave a comment and a link to the discussion here to explain why we need this?

Reply:
Done

Reply:
Can you put the discussion link here as well? https://mail.openjdk.org/pipermail/core-libs-dev/2024-November/133773.html Just in case it's resolved in a future JDK version, we should replace this with something else.

Reply:
Missed that, done now!

Reply:
Thanks!!